Integrations/EDL/EDL.py

```diff
@@ -224,8 +271,14 @@ def run_long_running(params):
         else:
             demisto.debug('Starting HTTP Server')
 
-        server = WSGIServer(('', port), APP, **ssl_args)
-        server.serve_forever()
+        server = WSGIServer(('', port), APP, **ssl_args, log=DEMISTO_LOGGER)
+        if is_test:
+            server_process = Process(target=server.serve_forever)
+            server_process.start()
+            time.sleep(10)
```

Comment:
Let's shorten this. Waiting 10 seconds is too much. Take this down to 5 seconds (should be sufficient).